capability_linux.go

```diff
@@ -358,11 +358,14 @@ func (c *capsV3) Apply(kind CapType) (err error) {
 			if c.Get(AMBIENT, i) {
 				action = pr_CAP_AMBIENT_RAISE
 			}
-			err := prctl(pr_CAP_AMBIENT, action, uintptr(i), 0, 0)
```

Comment:
To clarify: this `err` was not ever returned, so the code being removed by this commit is essentially equivalent to
```go
_ = prctl(pr_CAP_AMBIENT, action, uintptr(i), 0, 0)
```